app/views/lists/show.html.erb

```diff
@@ -11,10 +11,13 @@
 </p>
 
 <ul>
-  <% @list.papers.each do |paper| %>
-    <li>
-      <%=paper.title%>
-    </li>
+  <% @list.references.each do |reference| %>
+    <%if reference.paper.present?%>
+      <li>
+      <%=reference.paper.title%> 
+      <%= link_to 'Remove', reference_path(reference) %> 
```

Comment:
no, for some reason when I saw the link was showing up in show, I assumed
it worked, because I am living in a delusional world.

On Sun, Jul 10, 2016 at 12:49 PM, Brian Kung <notifications@github.com>
wrote:

> In app/views/lists/show.html.erb
> <https://github.com/briankung/projectcredo/pull/11#discussion_r70185799>:
>
> > @@ -11,10 +11,13 @@
> >  </p>
> >
> >  <ul>
> > -  <% @list.papers.each do |paper| %>
> > -    <li>
> > -      <%=paper.title%>
> > -    </li>
> > +  <% @list.references.each do |reference| %>
> > +    <%if reference.paper.present?%>
> > +      <li>
> > +      <%=reference.paper.title%>
> > +      <%= link_to 'Remove', reference_path(reference) %>
>
> Oh, gotcha, so it actually wasn't working before.
> … <#m_1330157832404823459_>
> On Sun, Jul 10, 2016 at 12:46 PM, wsf246 ***@***.***> wrote: In
> app/views/lists/show.html.erb <#11 (comment)
> <https://github.com/briankung/projectcredo/pull/11#discussion_r70185738>>:
> > @@ -11,10 +11,13 @@ > </p> > > <ul> > - <% @list.papers.each do |paper|
> %> > - <li> > - <%=paper.title%> > - </li> > + <% @list.references.each do
> |reference| %> > + <%if reference.paper.present?%> > + <li> > +
> <%=reference.paper.title%> > + <%= link_to 'Remove',
> reference_path(reference) %> ok, I updated it. It wasn't working before now
> it does. On Sun, Jul 10, 2016 at 12:41 PM, Brian Kung ***@***.***> wrote: …
> <#m_-6806643356669847570_> In app/views/lists/show.html.erb <#11
> <https://github.com/briankung/projectcredo/pull/11> (comment) <#11
> (comment)
> <https://github.com/briankung/projectcredo/pull/11#discussion_r70185642>>>:
> > @@ -11,10 +11,13 @@ > </p> > > <ul> > - <% @list.papers.each do |paper|
> %> > - <li> > - <%=paper.title%> > - </li> > + <% @list.references.each do
> |reference| %> > + <%if reference.paper.present?%> > + <li> > +
> <%=reference.paper.title%> > + <%= link_to 'Remove',
> reference_path(reference) %> Do you have it set up as `resources
> :references, only: [:create, :destroy]` ? I was thinking that if there's
> only one matching path for `reference_path(reference)` then it would
> default to `DELETE` but I'm not sure that actually makes sense. …
> <#m_1087154856280177285_> On Sun, Jul 10, 2016 at 12:35 PM, wsf246
> ***@***.***> wrote: In app/views/lists/show.html.erb <#11
> <https://github.com/briankung/projectcredo/pull/11> <#11
> <https://github.com/briankung/projectcredo/pull/11>> (comment) <#11
> <https://github.com/briankung/projectcredo/pull/11> (comment) <#11
> (comment)
> <https://github.com/briankung/projectcredo/pull/11#discussion_r70185530>>>>:
> > @@ -11,10 +11,13 @@ > </p> > > <ul> > - <% @list.papers.each do |paper|
> %> > - <li> > - <%=paper.title%> > - </li> > + <% @list.references.each do
> |reference| %> > + <%if reference.paper.present?%> > + <li> > +
> <%=reference.paper.title%> > + <%= link_to 'Remove',
> reference_path(reference) %> It does work and I don't know why. When I rake
> routes, I have 2 routes, one called references_path (which is linked to my
> create in my controller) and reference_path (which is my destroy.) I was
> confused as well as to why these got associated this way because I didn't
> explicitly set this anywhere that I can tell. … <#m_-5653474986531022354_>
> On Sun, Jul 10, 2016 at 12:32 PM, Brian Kung ***@***.***> wrote: In
> app/views/lists/show.html.erb <#11
> <https://github.com/briankung/projectcredo/pull/11> <#11
> <https://github.com/briankung/projectcredo/pull/11>> <#11
> <https://github.com/briankung/projectcredo/pull/11> <#11
> <https://github.com/briankung/projectcredo/pull/11>>> (comment) <#11
> <https://github.com/briankung/projectcredo/pull/11> <#11
> <https://github.com/briankung/projectcredo/pull/11>> (comment) <#11
> <https://github.com/briankung/projectcredo/pull/11> (comment) <#11
> (comment)
> <https://github.com/briankung/projectcredo/pull/11#discussion_r70185491>>>>>:
> > @@ -11,10 +11,13 @@ > </p> > > <ul> > - <% @list.papers.each do |paper|
> %> > - <li> > - <%=paper.title%> > - </li> > + <% @list.references.each do
> |reference| %> > + <%if reference.paper.present?%> > + <li> > +
> <%=reference.paper.title%> > + <%= link_to 'Remove',
> reference_path(reference) %> Does this work? And if so, why does it work?
> I'd expect you to need to send a method: :delete somewhere in there,
> depending on how the routes are wired. — You are receiving this because you
> authored the thread. Reply to this email directly, view it on GitHub <
> https://github.com/briankung/projectcredo/pull/11/files/67e353f1d94ac8392877ca0f2e76ee0b742ce933#r70185491>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/ACKsZPXu1mjHH_EeyX7bAJtAfTEmy2Irks5qUSzKgaJpZM4JI4Ma>
> . — You are receiving this because you commented. Reply to this email
> directly, view it on GitHub <
> https://github.com/briankung/projectcredo/pull/11/files/67e353f1d94ac8392877ca0f2e76ee0b742ce933#r70185530>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/ACtGx10JDDjvXfTKXxebYSBxdhW-00lRks5qUS1QgaJpZM4JI4Ma>
> . -- Brian Kung http://about.me/briankung/ <
> http://www.facebook.com/brian.kung> <http://www.linkedin.com/in/briankung>
> <http://www.twitter.com/briankung> BE A UNICORN! — You are receiving this
> because you authored the thread. Reply to this email directly, view it on
> GitHub <
> https://github.com/briankung/projectcredo/pull/11/files/67e353f1d94ac8392877ca0f2e76ee0b742ce933#r70185642>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/ACKsZMScNvVXVAQsSI_G6P4cSJ1PvBlkks5qUS6-gaJpZM4JI4Ma>
> . — You are receiving this because you commented. Reply to this email
> directly, view it on GitHub <
> https://github.com/briankung/projectcredo/pull/11/files/67e353f1d94ac8392877ca0f2e76ee0b742ce933#r70185738>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/ACtGx7NNsxfPPyxsk8cbjR9kOb9ffjdbks5qUS_2gaJpZM4JI4Ma>
> .
> -- Brian Kung http://about.me/briankung/ <
> http://www.facebook.com/brian.kung> <http://www.linkedin.com/in/briankung>
> <http://www.twitter.com/briankung> BE A UNICORN!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/11/files/67e353f1d94ac8392877ca0f2e76ee0b742ce933#r70185799>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACKsZMN-nUaarHNqBpVEFhJZ6k0M0iOpks5qUTC6gaJpZM4JI4Ma>
> .
>